hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -312,7 +312,7 @@
 
   public static final ConfigProperty<String> EMBEDDED_TIMELINE_SERVER_USE_ASYNC_ENABLE = ConfigProperty
       .key("hoodie.embed.timeline.server.async")
-      .defaultValue("false")
+      .defaultValue("true")
```

Comment:
Does the async timeline server has overlapping logic with metadata table ? If you user opens a metadata table, a better solution is reading the inc files from that.

Reply:
Async per se, is not tied to metadata table. It just turns on asynchronous handling of requests to timeline server. We benchmarked it to about 95K/sec.

Reply:
Wrt reading metadata table directly vs via timeline server, we cache certain things on the timeline server - like open file handles and even listing caches. So that can be efficient than reading entire table over and over from executors. (At least in spark)